ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/cards/MapCardAdapter.kt

```diff
@@ -154,7 +156,9 @@ class MapCardAdapter(
 
         coroutineScope.launch {
           val submissionCount = fetchSubmissionCount(loi)
-          submissions.text = LoiCardUtil.getSubmissionsText(submissionCount)
+          withContext(Dispatchers.Main) {
+            submissions.text = LoiCardUtil.getSubmissionsText(submissionCount)
```

Comment:
See https://github.com/google/ground-android/issues/2015#issuecomment-1785804529

Reply:
Replied in the same bug.